earth_enterprise/src/third_party/defusedxml/SConscript

```diff
@@ -0,0 +1,90 @@
+Import('third_party_env')
```

Comment:
We need to include the copyright header in this file. The Copyright should read "Copyright 2019 The Open GEE Contributors".